x/incentive/keeper/keeper.go

```diff
@@ -238,16 +236,20 @@ func (k Keeper) UpdateUncommittedTokens(ctx sdk.Context, epochIdentifier string,
 	k.UpdateTotalCommitmentInfo(ctx)
 
 	// Calculate 65% for LP, 35% for Stakers
-	dexRevenue := sdk.NewDecCoinsFromCoins(k.tci.TotalFeesCollected...)
-	devRevenue65 := dexRevenue.MulDecTruncate(sdk.NewDecWithPrec(65, 1))
-	devRevenue35 := dexRevenue.Sub(devRevenue65)
+	// Collect DEX revenue collected
+	dexRevenue := k.CollectDEXRevenusToIncentiveModule(ctx)
+
+	dexRevenueDec := sdk.NewDecCoinsFromCoins(dexRevenue...)
+	rewardPercentForLps := k.GetDEXRewardPercentForLPs(ctx)
+	devRevenueForLps := dexRevenueDec.MulDecTruncate(rewardPercentForLps)
+	devRevenueForStakers := dexRevenueDec.Sub(devRevenueForLps)
 
 	// Fund community pool based on the communtiy tax
-	devRevenueRemained35 := k.UpdateCommunityPool(ctx, devRevenue35)
+	devRevenueRemained35 := k.UpdateCommunityPool(ctx, devRevenueForStakers)
```

Comment:
Can you remove 35 suffix on the variable?

Reply:
Can you write reward split formula as comment? It does not have enough comments for this math operations